configs/evonet/core/dashd.conf

```diff
@@ -37,7 +37,7 @@ allowprivatenet=0
 
 printtoconsole=1
 
-[devnet]
```

Comment:
This should work as of 0.16 (https://dashcore.readme.io/docs/dash-core-wallet-configuration-file). It was backported from [Bitcoin](https://bitcoincore.org/en/releases/0.17.0/#configuration-sections-for-testnet-and-regtest)

Reply:
Well, this branch still uses 0.15

Reply:
We need to update it than